charts/fluentd-elasticsearch/templates/configmaps.yaml

```diff
@@ -472,12 +472,14 @@ data:
       @type elasticsearch
       @log_level info
       include_tag_key true
-      type_name _doc
       host "#{ENV['OUTPUT_HOST']}"
       port "#{ENV['OUTPUT_PORT']}"
       scheme "#{ENV['OUTPUT_SCHEME']}"
       ssl_verify "#{ENV['OUTPUT_SSL_VERIFY']}"
       ssl_version "#{ENV['OUTPUT_SSL_VERSION']}"
+{{- if (ne .Values.elasticsearch.typeName "") }}
```

Comment:
not sure if this makes sense as a default is set in the values.yaml and therfore its always true...

Reply:
In my use case, I have overridden the value with "". The logic above allows this to be omitted when the value is overridden with an empty string.

In this situation, the elasticsearch plugin defaults to type `fluentd`